models/cvecontents.go

```diff
@@ -313,11 +318,11 @@ var AllCveContetTypes = CveContentTypes{
 	SUSE,
 	DebianSecurityTracker,
 	WPVulnDB,
-	NodeSec,
-	PythonSec,
-	PhpSec,
-	RubySec,
-	RustSec,
+	// NodeSec,
```

Comment:
Does not Trivy need here?

Reply:
You can see it here.
https://github.com/future-architect/vuls/pull/971/files#diff-7d6bddda547d3a8f458448a93dad539dR284

Reply:
That line just defined const `Trivy`?
`AllCveContentTypes` does not have `Trivy`.

Maybe we need to set Trivy to `AllCveContentTypes` to show cve titles and so on.
https://github.com/future-architect/vuls/blob/0ff7641471ad115c801de8d5429af4ac237fc04d/models/vulninfos.go#L241

Reply:
Before the fix, I was downloading Trivy's DB in light mode and using it only for detection.
However, since Trivy's DB has Title, summary, refs, severity and may have information that NVD, OVAL, etc. do not have, I decided to get Trivy's information in full mode and display it in the report.